src/adapters/bidderFactory.js

```diff
@@ -493,19 +497,55 @@ export const processBidderRequests = hook('async', function (spec, bids, bidderR
         );
         break;
       case 'POST':
-        ajax(
-          request.url,
-          {
-            success: onSuccess,
-            error: onFailure
-          },
-          typeof request.data === 'string' ? request.data : JSON.stringify(request.data),
-          getOptions({
-            method: 'POST',
-            contentType: 'text/plain',
-            withCredentials: true
-          })
-        );
+        const enableGZipCompression = bidderSettings.get(spec.code, 'endpointCompression');
```

Comment:
I find this surprising, why would this be controlled by the publisher? Would they know or care whether your server can accept gzip, and would they have a reason to change it?

I would have made this a flag in `options`.

Reply:
The latter IMO. I don't see the advantage of a separate file and I don't think you can remove the burden from bidders anyway,  they'll need server side changes.

Reply:
agreed: 'OR expect each bidder to send appropriate flag value in request.options?"

Reply:
@dgirardi , makes sense, will move the feature-flag from bidderSettings to request.options !